internal/configunmarshaler/defaultunmarshaler.go

```diff
@@ -75,17 +81,17 @@ type configSettings struct {
 	Service    map[string]interface{}                        `mapstructure:"service"`
 }
 
-type defaultUnmarshaler struct{}
+type DefaultUnmarshaler struct{}
```

Comment:
Is this used outside of this package?

Reply:
Nope, initially I did not define the interface in the internal and I had to return it. But I figured that I have to define the interface since it is expose as argument.